README.md

```diff
@@ -1,4 +1,4 @@
-# Fast Downward
+<img src="misc/images/fast-downward.png" width="800" alt="Fast Downward">
```

Comment:
On mobile the image size is perfect, on desktop the image is quite large. If something adapting is possible that could be nice, but I can live with it as it is.

Reply:
I remember that we had the convention modules names are written with underscores, scripts with hyphen. What is our convention on other file types?
Now that you write `fast-downward.png` with a hyphen I am a bit worried that people will copy the spelling from this (at the same time there is a hyphen in our url and it is more likely that people copy the name from there).

Reply:
I don't think many people will look at the file name. I did not find something analogous in the repo, so I took the hyphen over from other files. Regarding the image size, I had a look what other projects do. I also found it quite large at the beginning, but to stick out in the github GUI I actually makes sense.